doc/src/manual/multi-threading.md

```diff
@@ -337,6 +339,32 @@ julia> begin
 All three options are equivalent. Note how the final version requires an explicit `try`-block to ensure that the lock is always unlocked, whereas the first two version do this internally. One should always use the lock pattern above when changing data (such as assigning
 to a global or closure variable) accessed by other threads. Failing to do this could have unforeseen and serious consequences.
 
+#### [Using Base.Lockable to associate a lock and a value](@id man-lockable)
+As mentioned in the previous section, the helper-type [`Base.Lockable`](@ref) can be used to programmatically ensure the association between a lock and a value. This is generally recommended, as it is both less prone to error and more readable for others compared to having the association only by convention.
+
+
+```julia-repl
+julia> my_array = [];                   # Simple empty array
```

Comment:
Some comments about this code block:

- I think it is overcommenting a bit. There is not really a need to comment on creating an empty array.
- The comments become quite long on a single line, which will require horizontal scrolling to read on the website.


I'd take some inspiration of how the section below is written, ideally the text before the code should be enough to understand the code.